Code/Framework/AzFramework/AzFramework/Physics/Common/PhysicsTypes.h

```diff
@@ -24,11 +24,12 @@ namespace Physics
 
 namespace AzPhysics
 {
-    //! Default Scene Names and Crc32
-    static constexpr const char* DefaultPhysicsSceneName = "DefaultScene";
-    static constexpr const AZ::Crc32 DefaultPhysicsSceneId = AZ_CRC_CE(DefaultPhysicsSceneName);
-    static constexpr const char* EditorPhysicsSceneName = "EditorScene";
-    static constexpr const AZ::Crc32 EditorPhysicsSceneId = AZ_CRC_CE(EditorPhysicsSceneName);
+    //! Maximum number of scenes, Default Scene Names and Crc32
+    constexpr AZ::u32 MaxNumberOfScenes = 64;
```

Comment:
Is it worth adding a comment explaining the limitation on Android which limits this?

Reply:
I don't think so, very platform specific...